conda/recipes/librmm/meta.yaml

```diff
@@ -60,12 +67,20 @@ outputs:
       run_exports:
         - {{ pin_subpackage("librmm", max_pin="x.x") }}
       ignore_run_exports_from:
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }}
+        {% else %}
         - {{ compiler('cuda') }}
+        {% endif %}
     requirements:
       build:
         - cmake {{ cmake_version }}
```

Comment:
This is another split-recipe question, I think. Do we need the compiler here _and_ in the top-level build requirements?

Reply:
In my understanding, only `cmake` is required because the script is purely a copy operation: https://github.com/rapidsai/rmm/blob/3c40cbccb2291288cab8e924c734de9a2497ac8c/conda/recipes/librmm/install_librmm.sh#L3